src/main/scala/com/atomist/rug/function/github/CreateTagFunction.scala

```diff
@@ -35,15 +31,14 @@ class CreateTagFunction
     logger.warn(s"Invoking createTag with tag '$tag', message '$message', sha '$sha', owner '$owner', repo '$repo', apiUrl '$apiUrl' and token '${safeToken(token)}'")
 
     val apiUrlSlash = if (apiUrl.endsWith("/")) apiUrl else s"$apiUrl/"
```

Comment:
So are you going to centralize it?

This was added because previously the code assumed `apiUrl` did not end with a forward slash and always inserted one. The convention the platform uses is that the stored API URL, and hence the one delivered to this and all other functions, does have a forward slash. This resulted in the URL looking like "https://api.github.com//issues...", and GitHub did not like the double forward slash.